JSDemos/Demos/Scheduler/GroupOrientation/description.md

```diff
@@ -1 +1,5 @@
-This example shows the **workWeek** view with different group orientations. Select the *"Vertical Grouping"* tab to&nbsp;see the resource headers organized in&nbsp;a&nbsp;column and the *"Horizontal Grouping"*&nbsp;&mdash; arranged in&nbsp;a&nbsp;row.
\ No newline at end of file
+[Resources](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/) are categories used to classify appointments. To understand what resources are, consider the following example: in an educational center lectures are held in several rooms; in Scheduler terms, room is a resource kind, individual rooms are resource instances, and lectures are appointments that use these resource instances.
```

Comment:
We do need to make a connection between grouping and resources. However, this demo is about grouping, and we shouldn't dive deep into explaining the resources concept. The following description would suffice:

```suggestion
Appointments can be grouped by resources. This demo declares a single resource&mdash;Priority. Refer to the following demo for more information about resources and their configuration: [Resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).
```